### PR TITLE
#153 Adding documentation on how to create entities from memorious

### PR DESCRIPTION
**Pull request by Rosencrantz**

### CODE REVIEW

File: docs/buildingcrawler.md [hunk 1]
```diff
@@ -344,12 +344,40 @@ And optionally:
...
+      and:
+        - mime_group: web
+    properties:
+      title: .//meta[@property="og:title"]/@content
+      author: .//meta[@name="author"]/@content
+      publishedAt: .//*[@class="date"]/text()
+      description: .//meta[@property="og:description"]/@content
+```
+
+The `data` `alpeh_emit_entity` emits to the next stages includes the following new items:
```
Comment (sunu) [inline docs/buildingcrawler.md:376]: May be it's worth adding this as the next stage in the example yaml code section above as well?

Somewhat related, the example config we have doesn't look right (the comment and the param seem to be unrelated): https://github.com/alephdata/memorious/blob/master/example/config/simple_article_scraper.yml#L52

Comment (Rosencrantz) [reply docs/buildingcrawler.md:376]: Yeah, that seems fair.

**Review by sunu — APPROVED** [review]
